src/Brick/Widgets/ProgressBar.hs

```diff
@@ -10,7 +10,6 @@ module Brick.Widgets.ProgressBar
 where
 
 import Lens.Micro ((^.))
-import Data.List (splitAt)
```

Comment:
Unrelated to language fixes - just fixing an unused import warning (splitAt is exported from Prelude)